addons/common/functions/fnc_findUnloadPosition.sqf

```diff
@@ -67,9 +67,9 @@ private _originAGL = ASLtoAGL _originASL;
 
 //Do a manual search for empty pos (handles underwater, buildings or piers)
 TRACE_2("Checking for unload",_originAGL,_radiusOfItem);
-private _rangeToCheck = 0;
+private _rangeToCheck = 25;
 while {_rangeToCheck < _maxDistance} do {
-    private _roundDistance = random _rangeToCheck;
+    private _roundDistance = (random _rangeToCheck) max 15;
```

Comment:
15m is huge for most things, it used to unload a object pretty much at your feet

Reply:
How to handle unloading MBTs or something bigger than 15m then?

Reply:
No tank is longer than 10 meters.